includes/rest-api/CMB2_REST.php

```diff
@@ -117,6 +124,10 @@ public static function maybe_init_and_hookup( CMB2 $cmb ) {
 	public function __construct( CMB2 $cmb ) {
 		$this->cmb = $cmb;
 		self::$boxes[ $cmb->cmb_id ] = $this;
+		self::$rest_slug = 'cmb2';
```

Comment:
@Mte90 What's the point of this line (127) when we have a predefined value on line 81?

Reply:
I saw that in case of multiple metabox where the parameter is not defined without that line, in other rest pages where should be the default value was replaced.
With forcing a reset the code was working and keeping backward compatibility.